Lib9c/Action/TransferAsset.cs

```diff
@@ -9,21 +9,32 @@
 using System.Linq;
 using System.Runtime.Serialization;
 using Lib9c.Abstractions;
+using Nekoyume.Helper;
 using Nekoyume.Model;
 using Serilog;
 
 namespace Nekoyume.Action
 {
     /// <summary>
     /// Hard forked at https://github.com/planetarium/lib9c/pull/636
-    /// Updated at https://github.com/planetarium/lib9c/pull/957
+    /// Updated at https://github.com/planetarium/lib9c/pull/1718
     /// </summary>
     [Serializable]
     [ActionType("transfer_asset3")]
     public class TransferAsset : ActionBase, ISerializable, ITransferAsset, ITransferAssetV1
     {
         private const int MemoMaxLength = 80;
 
+        public const long WhiteListAvailableIndex = 6_200_000L;
+
+        public static readonly IReadOnlyList<Address> WhiteList = new List<Address>
```

Comment:
```suggestion
        public static readonly IReadOnlyList<Address> AllowedCrystalTransferers = new List<Address>
```

Reply:
Is there any external resource(e.g., NCIP, DAO proposal) to justify their addresses? @ipdae @jaeho0103